packages/urql/updates/makeAppendToPaginatedSetResolver.js

```diff
@@ -0,0 +1,110 @@
+import { PAGINATION_ARGS } from '../resolvers/makePaginationResolver'
+
+/*
```

Comment:
Love this, thank you

Reply:
Thank you for seeing me :) OMG I sweat blood to get to to that, and it indeed solves most of the URQL woes to this point. Both in understanding how to get there, and in its actual utility.